waku/v2/node/config.nim

```diff
@@ -75,6 +75,11 @@ type
       defaultValue: false
       name: "rln-relay" }: bool
     
+    rlnRelayMemIndex* {.
+      desc: "the index of node in the rln-relay group: a value between 0-49 inclusive",
```

Comment:
Presumably this is temporary, so might want to indicate this with something like `(for development)` `(experimental)` or so

Reply:
Thanks, done!